hyco-https/tests/pipe.test.js

```diff
@@ -0,0 +1,138 @@
+var https = require('..');
+const Stream = require('stream');
+
+var ns = process.env.SB_HC_NAMESPACE ? process.env.SB_HC_NAMESPACE.replace(/^"(.*)"$/, '$1') : null;
+var path = process.env.SB_HC_PATH ? process.env.SB_HC_PATH : "a2";
+var keyrule = process.env.SB_HC_KEYRULE ? process.env.SB_HC_KEYRULE.replace(/^"(.*)"$/, '$1') : null;
+var key = process.env.SB_HC_KEY ? process.env.SB_HC_KEY.replace(/^"(.*)"$/, '$1') : null;
+
+expect(ns).toBeDefined();
+expect(path).toBeDefined();
+expect(keyrule).toBeDefined();
+expect(key).toBeDefined();
+
+var smallMessage = "SmallMessage";
+var kb = "";
+for (var i = 1024; i >= 0; i--) {
```

Comment:
>(var i = 1024; i >= 0; i-- [](start = 4, length = 26)

Does this do 1025?

Reply:
intentional to exceed 64kb

---
In reply to: [340269407](https://github.com/Azure/azure-relay-node/pull/44#discussion_r340269407) [](ancestors = 340269407)

Reply:
I guess naming it "kb" though it's kbPlusOne confused me.

---
In reply to: [340272075](https://github.com/Azure/azure-relay-node/pull/44#discussion_r340272075) [](ancestors = 340272075,340269407)

Reply:
Good point, will change it

---
In reply to: [340272075](https://github.com/Azure/azure-relay-node/pull/44#discussion_r340272075) [](ancestors = 340272075,340269407)